governing-documents/security.md

```diff
@@ -0,0 +1,274 @@
+fixes quickly. Approval is not given lightly: project leadership
+(maintainers) must be convinced that institutions on the list **“need to know"**
+about issues in advance.
+
+Participation in an embargo list should not be taken lightly. List members
+are expected to respect the materials shared through it and not disclose any
+information to unauthorized parties until the public disclosure date.
+Institutions are on this list because their presence helps the project and its
+users; if their actions do not help the project and its users, they can expect
+to be removed from the list.
+
+The `XYZ` project team will publish guidelines for participation in the
+embargo list. Alternatively, send an email to [Hyperledger Foundation security
+mailing list](mailto:security@lists.hyperledger.org) with the reason for
+you to be part of the embargo list.
+
+`If the project does not maintain an embargo list.`
+This project does not maintain an embargo list. This is because `we are too
+small or there are no known large users of this project–if you are one, let us
+know!`, and may change if `people start using our software in production`.
+
+`If the project does maintain an embargo list.`
+This project maintains a embargo list.  The security team listed above is
+responsible for maintaining the embargo list.  Members of the security team
+will send notifications of security issues to the embargo list using the
+[OpenSSF embargo format](https://github.com/ossf/oss-vulnerability-guide/blob/main/templates/notifications/embargo.md).
+
+The list itself is private in order to make it slightly more difficult for
+attackers with vulnerabilities to find systems to attack. Entities may be
+added to the embargo list by a majority vote of the `project` security response
+team and should request to join the embargo list by contacting one or more of
+the members of the security response team. If there is an issue about embargo
+list membership where an entity feels like they are being dealt with unfairly
+by the security response team, then they are encouraged to bring up the issue
+in front of the Hyperledger TOC, who can act as moderators.
+
+## (GitHub) Security Advisories
+
+It is highly recommended that Hyperledger Foundation projects use [GitHub
+security advisories and the GitHub security process](https://docs.github.com/en/code-security/security-advisories) for
+handling security vulnerabilities. In particular, this is strongly recommended
+for projects that do not have a large number of security experts as the
+features serve as a nice set of **guardrails** to help make sure that things are
+done correctly.
+
+This project `does/does not` use GitHub security advisories. `More precisely,
+describe what features (if any) are used here if the whole package is not
+utilized.`
+
+`If the project does not use GitHub security advisories, explain how security
+issues are publicly disclosed.`
+
+## Private Patch Deployment Infrastructure
```

Comment:
Maintaining a private repo for the long term seems like a lot of overhead. Would it be better to document how to create a private repo for when it is needed?  Not sure exactly how to do that, but I’m thinking this could be generic guidance, that a project could dry run and customize for their repo if deemed important to have ready ahead of time.  E.g. create private fork of repo, change X, Y, Z in the pipeline to produce private artifacts.

Reply:
@swcurran this is done automagically by GitHub on a per-patch basis.